### ISSUE TITLE
Eip - Allow to reserve static ip

### ISSUE THREAD

**Issue by tanandy** [ovh/public-cloud-roadmap#84]
As a user I want to be able to reserve/attach/detach/release a static external ip
I want to be able to consume this feature through standard APIs / OVHcloud Customer panel and attach it to different cloud component (Lb, instance, ...)
Then i will be able to keep my reserved EIP even if my load balancer, instance ... is lost or deleted

**Comment by tanandy** [comment]
@mhurtrel any news on it ? its a must have

**Comment by mhurtrel** [comment]
I agree with the need and think it is linked to https://github.com/ovh/public-cloud-roadmap/issues/40 (floating IPs) my colleague @JacquesMrz  can give you ETA details

**Comment by tanandy** [comment]
ok hope it is the same thing :) thank you

**Comment by antonin-a** [comment]
Closed as solved by Floating IPs : https://www.ovhcloud.com/fr/public-cloud/floating-ip/
